From 5d91a66431c0552317eb3dbd7796c2d8cf6fad97 Mon Sep 17 00:00:00 2001
From: Ken Ma <make@marvell.com>
Date: Thu, 18 May 2017 14:50:03 +0800
Subject: [PATCH 0720/1200] arm64: dts: a3700: set lane2 comphy speed to 6Gbps

Lane2 is used for SATA, its phy speed is auto-negotiated at 6Gbps;
the lane2 phy speed in dts is synchronized to 6Gbps so that boot log
can print out correct lane2 phy speed.

Change-Id: I0815286db989b391e5015c17bbb61d60d076eeb1
Signed-off-by: Ken Ma <make@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/54456
Reviewed-by: Igal Liberman <igall@marvell.com>
Tested-by: Igal Liberman <igall@marvell.com>
---
 arch/arm/dts/armada-3720-db.dts          | 2 +-
 arch/arm/dts/armada-3720-espressobin.dts | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/armada-3720-db.dts b/arch/arm/dts/armada-3720-db.dts
index 135f1f21ea..5bcf06c878 100644
--- a/arch/arm/dts/armada-3720-db.dts
+++ b/arch/arm/dts/armada-3720-db.dts
@@ -113,7 +113,7 @@
 
 	phy2 {
 		phy-type = <COMPHY_TYPE_SATA0>;
-		phy-speed = <COMPHY_SPEED_5G>;
+		phy-speed = <COMPHY_SPEED_6G>;
 		phy-invert = <COMPHY_POLARITY_NO_INVERT>;
 	};
 };
diff --git a/arch/arm/dts/armada-3720-espressobin.dts b/arch/arm/dts/armada-3720-espressobin.dts
index 735e8d376b..650b811daa 100644
--- a/arch/arm/dts/armada-3720-espressobin.dts
+++ b/arch/arm/dts/armada-3720-espressobin.dts
@@ -83,7 +83,7 @@
 
 	phy2 {
 		phy-type = <COMPHY_TYPE_SATA0>;
-		phy-speed = <COMPHY_SPEED_5G>;
+		phy-speed = <COMPHY_SPEED_6G>;
 	};
 };
 
-- 
2.22.0

